README.md

```diff
@@ -8,9 +8,44 @@ See [fax/constrained/constrained_test.py](fax/constrained/constrained_test.py) f
 
 ## Installation
 
+
+To get the latest version from Github: 
+```sh
+pip install git+https://github.com/gehring/fax.git
+```
+
+Otherwise on PyPI:
 ```sh
 pip install jax-fixedpoint
 ```
+## Basic Usage
+The main entry point for Christianson's two-phases reverse accumulation is through `fax.implicit.two_phase_solver`. For example, imagine that you have a [fixed-point iteration method](https://en.wikipedia.org/wiki/Fixed-point_iteration) like [Power iteration](https://en.wikipedia.org/wiki/Power_iteration) and want to compute the gradient of a function of its output. You could write something like: 
+```python
+from functools import partial
+import jax
+import jax.numpy as jnp
+from fax import implicit
+
+
+def make_power_iteration(A):
+  def _power_iteration(b):
+    b = A @ b
+    return b/jnp.linalg.norm(b)
+  return _power_iteration
+
+def make_objective(A):
+  b0 = jnp.ones((A.shape[0]))
+  power_iteration = partial(implicit.two_phase_solve, make_power_iteration, b0)
+  def _objective(A):
+    b = power_iteration(A)
+    return (b.T @ A @ b)/(b.T @ b)
+  return _objective
```

Comment:
This is a great example!

I have to admit that I find the functions returning functions to be a bit more confusing than plain objects.  It might just be my bias.

Also, just to keep in mind that your b0 is an example of what we were talking about in chat.  If someone tries to pass in b0 as another parameter to `make_objective`, and that b0 were a tracer, then it would leak.  I'm not suggesting any changes though, I like the simplicity of the code when illustrating "basic usage".

Reply:
I wanted to stay close to what was originally there. We could change it to better reflect how we think people should call it. WDYT?

Reply:
Yeah, it makes sense to me to reflect how you think people should call it.

Reply:
Updated and expanded a bit.